tests/linalg_test.py

```diff
@@ -1564,5 +1564,11 @@ def testSchurBatching(self, shape, dtype):
       Ts, Ss = vmap(lax.linalg.schur)(args)
       self.assertAllClose(reconstruct(Ss, Ts), args, atol=1e-4)
 
+  @parameterized.parameters(0, 1, 5)
```

Comment:
`n=0` and `n=1` as corner cases, and if it works for `n=5` then it must be correct